frontend/src/views/web/analytics/host-analytics.html

```diff
@@ -28,7 +28,8 @@ <h4 class="fs-18">Select challenge to view Analysis </h4>
         </div>
         <div class="col s12 m7">
             <h4 class="fs-18">Challenge Analysis</h4>
-            <div class="main-title analytics-challenge-title" ng-if="!analytics.isTeamSelected">Please select challenge for analysis.</div>
+            <div class="main-title analytics-challenge-title" ng-if="!analytics.isTeamSelected && analytics.challengeListCount > 0">Please select challenge for analysis.</div>
+            <div class="main-title analytics-challenge-title" ng-if="!(analytics.challengeListCount > 0)">There are no challenges to analyze. Click <a ui-sref="web.challenge-host-teams">here</a> to create a new challenge.</div>
```

Comment:
!(a > b) => a < b   
try to use this expression and also cross check the value of `challengeListCount` from controller too.

Reply:
@aka-jain Made the changes.